### ISSUE TITLE
Show word edit history

### ISSUE THREAD

**Issue by maheshmnj** [maheshmnj/vocabhub#13]
When a word is edited, Its history must be saved to understand who made the edits.  

**Describe the solution you'd like**

## Frontend

A word detail must show 
 - last edit time and name
 - clicking the name should show the edits in ascending/descending order in the form of timeline

## Backend
We could maintain an edit history table containing edit entries and each history (entry) Id, must be recorded in a words table.
so basically each word must have an array or map of edits Ids.

**Comment by maheshmnj** [comment]
v0.5.9 All edits on that word are shown as of today

- [ ] Show only Approved edits

**Comment by maheshmnj** [comment]
fixed in v0.6.0 with diff visulaizer
